docs/test-pages/TCD-002.adoc

```diff
@@ -0,0 +1,33 @@
+= TCD-002
+
+== Purpose
+
+Checking that the IDE knows the jsl file extension as a JSL file.
+
+=== Tested requirements
+
+REQ-SYNT-001, REQ-MDL-001
+
+=== Used JSL model
+
+xref:resources/m1.jsl[m1.jsl], xref:resources/m1.txt[m1.txt]
+
+== Test scenario
+
+. Start the IDE (Eclipse or idea).
+
+. Open the xref:resources/m1.txt[m1.txt].
+
+. Verify that the IDE doesn't colour the text, because it is handle the file as a simple text file.
```

Comment:
'colour' => 'color' - We use American English grammar.
'handle' => 'handling' or 'it handles' (however I would use recognize here as well)

Reply:
Thank you!